zuul.d/project-templates.yaml

```diff
@@ -76,6 +76,23 @@
       The default set of functional test jobs for the OpenStack Charms
     check:
       jobs:
+        - mantic-bobcat:
+            voting: false
+            branches:
+              - master
+              - stable/2023.2
+              - stable/1.8
+              - stable/23.03
```

Comment:
ovn would be stable/23.09 , upstream has a 3 months cadence, the charms align with the March and September releases.

Reply:
Thanks, I've added a new commit for this